eventsources/sources/resource/start.go

```diff
@@ -135,14 +135,25 @@ func (el *EventListener) StartListening(ctx context.Context, dispatch func([]byt
 					continue
 				}
 
```

Comment:
`json.Marshal` on line 132 should be `json.Marshal(event.Obj)

Reply:
good catch. fixed